sourcetools/CMakeLists.txt

```diff
@@ -29,6 +29,13 @@ project(j9_sourcetools VERSION 1.0)
 
 set(CMAKE_JAVA_TARGET_OUTPUT_DIR "${CMAKE_CURRENT_BINARY_DIR}")
 
+# Due to the way cmake generates the java class path, it will generate a leading path separator.
+# This prevents fixpath from JDK15+ from properly translating the path.
+# To work around the issue we tack on a dummy path at the beginning of the output classpath.
+if((WIN32 AND NOT CMAKE_HOST_WIN32) AND (NOT JAVA_SPEC_VERSION  LESS 15))
```

Comment:
nit: double-space before `LESS`.

Rather than refer to a version that's already out of support, I think
```
  NOT JAVA_SPEC_VERSION  LESS 15
```
would be clearer expressed as
```
JAVA_SPEC_VERSION GREATER 11
```
The comment should be updated accordingly.